app/controllers/submit.js

```diff
@@ -704,7 +722,7 @@ export default Controller.extend(Analytics, BasicsValidations, COIValidations, N
             });
             // Closes section, so all panels closed if Upload section revisited
             this.get('panelActions').close('uploadNewFile');
-            this.send('next', this.get('_names.2'));
+            this.send('next', this.get('_names.1'));
```

Comment:
After we finish upload, we should go to the next section, meaning that we should call `send()` action with current panel name, which is `_names.1`, as parameter. This was `_names.2` because we've added the `Assertions` panel name in between `File` and `Basics` but the `Assertions` panel was not in place yet.